waku/node/v2/waku_types.nim

```diff
@@ -144,6 +144,8 @@ type
     listenStr*: string
     #multiaddrStrings*: seq[string]
 
+  WakuResult*[T] = Result[T, cstring]
```

Comment:
I'd much prefer an enum

Reply:
It is not needed at the moment as no different action is taken based on the error outcome, see chat example.

If this turns out to be needed for the applications that use it, we can adjust (or have an additional one that holds an enum for specific API calls).